src/Cli/dotnet/commands/dotnet-workload/config/WorkloadConfigCommandParser.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using Microsoft.DotNet.Workloads.Workload.Config;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class WorkloadConfigCommandParser
+    {
+        //  dotnet workload config --update-mode workload-set
+
+        public static readonly string UpdateMode_WorkloadSet = "workload-set";
```

Comment:
nit: typing - is hard. workloadset?

Also, I thought we were trying to not include "workload set" terminology? Not that I have a better proposal in this case...

Reply:
I'm pretty sure the guidance I got from @baronfel was to use dashes to separate words.  I think that's the style we're going with in the CLI.